lvr_ros/PolygonMesh[] polymeshes
string frame
---
lvr_ros/PolygonMesh polymesh
